serverjs/updatecards.js

```diff
@@ -233,6 +244,55 @@ function addTokens(card) {
         }
       }
     }
+    if (_catalog.dict[card.id].oracle_text.includes('Ascend (')) {
+      mentionedTokens.push({
+        tokenId: getTokenIDForSpecialCaseToken("City's Blessing"),
+        sourceCardId: _catalog.dict[card.id]._id
+      });
+    }
+    if (_catalog.dict[card.id].oracle_text.includes('Infect (')) {
```

Comment:
Does scryfall/oracle add the reminder text to all static abilities? They aren't printed on every card. Also, do you handle other sources of poison counters like Caress of Phyrexia?

Reply:
True, I changed the condition to "poison counter" since that is the way they are always talked about in the rules text. So this should cover all poison effects, even those without Infect.

Reply:
Oh, and Ascend is never mentioned without reminder text.